src/backend/function/Fusion.Resources.Functions/Functions/Notifications/ScheduledReportTimerTriggerFunction.cs

```diff
@@ -32,7 +32,7 @@ public class ScheduledReportTimerTriggerFunction
 
     [FunctionName("scheduled-report-timer-trigger-function")]
     public async Task RunAsync(
-        [TimerTrigger("0 0 8 * * MON", RunOnStartup = false)]
+        [TimerTrigger("0 0 6 * * MON", RunOnStartup = false)]
```

Comment:
Would it be better to start it at 5 so that we process it when the system is unused by users.